src/main/java/de/tum/in/www1/artemis/service/compass/CompassCalculationEngine.java

```diff
@@ -267,6 +269,12 @@ public void markModelAsUnassessed(long modelSubmissionId) {
                 continue;
             }
 
+            double elementConfidence = getConfidenceForElement(jsonElementID, modelId);
```

Comment:
please add some documentation to the method (JavaDoc) and into the code itself

Reply:
The JavaDoc for this method can be found in the CalculationEngine interface. I extended it a bit and added some comments to the code.